src/routes/lib/v1/PackageRequest.js

```diff
@@ -125,6 +128,25 @@ class PackageRequest extends BaseRequest {
 		return listing;
 	}
 
+	async getEntrypoints () {
+		let props = { type: this.params.type, name: this.params.name, version: this.params.version };
+		let packageEntrypoints = await PackageEntrypoints.find(props);
+
+		if (packageEntrypoints) {
+			return JSON.parse(packageEntrypoints.entrypoints);
+		}
+
+		let { entrypoints } = await jsDelivrRemoteService.usingContext(this.ctx).listResolvedEntries(this.params.type, this.params.name, this.params.version);
```

Comment:
This may need similar version validation as in `fetchFiles()` to ensure we don't cache semver ranges.

Reply:
Should we store the entrypoints/listing in the db even if the requested version doesn't match? it could safe us few requests to the origin in cost of the DB size.

Reply:
No, the idea is to also have "clean" data in the DB.